pnpm/package.json

```diff
@@ -58,7 +58,7 @@
   },
   "unpkg": "dist/pnpm.mjs",
   "scripts": {
-    "bundle": "pnpm --config.useNodeVersion=24.6.0 exec ts-node bundle.ts",
+    "bundle": "pnpm --config.useNodeVersion=24.6.0 node bundle.ts",
```

Comment:
This is about 1.1x faster.

<img width="907" height="593" alt="Image" src="https://github.com/user-attachments/assets/73c14a60-bbbf-401f-8810-55a33dce0cf5" />

That's not as much as I was hoping for, but I think it's significant when account for how many times this is ran on CI and locally.